pages/app/settings.vue

```diff
@@ -51,16 +51,22 @@
             </div>
 
             <div class="col-span-4 space-y-3">
-              <div class="relative h-56 aspect-square bg-gray-800 rounded-lg">
-                <div class="absolute inset-0">
-                  <input ref="profilePhotoInput" type="file" accept="image/*" class="hidden" @change="onSelectProfilePhoto" />
-                </div>
-                <div class="flex items-center justify-center h-full">
+              <div class="relative h-56 aspect-square bg-gray-800 rounded-lg overflow-hidden">
+                <input ref="profilePhotoInput" type="file" accept="image/*" class="hidden" @change="onSelectProfilePhoto" />
+                
+                <img 
+                  v-if="imagePreview || storeUser.getProfile?.profilePicPath" 
+                  :src="imagePreview || storeUser.getProfilePicBase64ByUserId(storeUser.getProfile?.id)" 
```

Comment:
How about we initialize `imagePreview` with getter data `getProfilePicBase64ByUserId`? You are already calling `fetchProfilePic`, so you can resolve the promise to initialize the `imagePreview`.